docusaurus/docs/publish-a-plugin/sign-a-plugin.md

```diff
@@ -60,27 +63,27 @@ Public plugins need to be reviewed by the Grafana team before you can sign them.
 
 1. Submit your plugin for [review](./publish-or-update-a-plugin.md).
 1. If we approve your plugin, you're granted a plugin signature level. You need this signature level to proceed.
-1. In your plugin directory, sign the plugin with the Access Policy token you just created. The Grafana sign-plugin tool creates a [MANIFEST.txt](#plugin-manifest) file in the `dist` directory of your plugin:
+1. In your plugin directory, sign the plugin with the Access Policy token you just created. The Grafana sign-plugin tool creates a [MANIFEST.txt](#add-a-plugin-manifest-for-verification) file in the `dist` directory of your plugin:
 
-   ```bash
+   ```shell npm2yarn
    export GRAFANA_ACCESS_POLICY_TOKEN=<YOUR_ACCESS_POLICY_TOKEN>
-   npx @grafana/sign-plugin@latest
+   npm @grafana/sign-plugin@latest
    ```
 
 ## Sign a private plugin
 
-1. In your plugin directory, sign the plugin with the Access Policy token you just created. The Grafana sign-plugin tool creates a [MANIFEST.txt](#plugin-manifest) file in the `dist` directory of your plugin.
+1. In your plugin directory, sign the plugin with the Access Policy token you just created. The Grafana sign-plugin tool creates a [MANIFEST.txt](#add-a-plugin-manifest-for-verification) file in the `dist` directory of your plugin.
 
-   ```bash
+   ```shell npm2yarn
```

Comment:
Please see my other comment related to `npm2yarn` not converting these code snippets correctly.